setup/install/providers/azure.md

```diff
@@ -79,11 +85,19 @@ hal config provider azure account add my-azure-account \
   --subscription-id $SUBSCRIPTION_ID \
   --default-key-vault $VAULT_NAME \
   --default-resource-group $RESOURCE_GROUP \
+  --packer-resource-group $RESOURCE_GROUP \
```

Comment:
Do we assume the user is using Packer? The reference doc for this command says "The resource group to use if baking images with Packer."

The doc also has " `` packer-storage-account`: The storage account to use if baking images with Packer." So, if we assume the user is using Packer, should we include this too?

Reply:
Good catch! Firstly we do assume the user is using Packer. Then, the parameter "--packer-storage-account" is out of date, which means it is not expected to appear in the ref doc any more and only the parameter "--packer-resource-group" is enough. I'll initial another PR to Halyard to fix this.

Reply:
Works for me